sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/CliSuite.scala

```diff
@@ -69,8 +69,11 @@ class CliSuite extends FunSuite with BeforeAndAfterAll with Logging {
       }
     }
 
+    // Propagate the current class path to the child to support *-provided profiles.
+    val extraEnv = Seq("SPARK_TEST_PARENT_CLASS_PATH" -> sys.props("java.class.path"))
```

Comment:
Rather than setting an env var here, could we just invoke the cli with the correct flags (driver-class-path, etc)? We went through a lot of care to make sure that the CLI, JDBC server, etc supported the full range of spark options. Given that, I wonder if we can do this using the existing options instead of introducing new env vars.